### PR TITLE
Some layout and applications of basic vector math

### PR DESCRIPTION
**Pull request by kwichmann**
There's now three sections (layouted by some basic Bootstrap):

Finding similar words.
Finding words between two other words.
Finding words that is to X as A is to B.

### CODE REVIEW

**Comment by shiffman** [comment]
I appreciate the effort in making it look nicer, but I try to keep my examples void of extra styling and design just to emphasize the basic functionality and to leave them blank for students to visually augment. I'm happy to merge and adjust if you prefer! Thank you so much for this!

File: word2vec/sketch.js [hunk 1]
```diff
@@ -7,34 +7,71 @@ let wordVecs;
...
   button.mousePressed(() => {
-    word = wordInput.value();
-    let nearest = Word2Vec.nearest(word, 10);
-    if (!nearest) {
-      results.html('no word vector found');
+    let word = wordInput.value();
+    results.html(findNearest(word, 10));
+  });
+
+  button2.mousePressed(() => {
```
Comment (shiffman) [inline word2vec/sketch.js]: maybe rename buttons to be descriptive rather than numbered? `buttonNearest`, `buttonAdd`, etc.?

Comment (kwichmann) [reply word2vec/sketch.js]: I definitely thought about that while making this. Will take a look at it.

**Comment by kwichmann** [comment]
You just do whatever works the best for you, Dan.

**Comment by shiffman** [comment]
will do, thank you!!

**Pull request merged**